patch/gfx9/chip/llpcGfx9Chip.cpp

```diff
@@ -30,6 +30,8 @@
  */
 #define DEBUG_TYPE "llpc-gfx9-chip"
 
+#include <stdio.h>
+
```

Comment:
Why is that required?

Reply:
Oh, this one was for the calls to `snprintf`.  Interestingly, it only happens with our internal build system (clang-based).  I didn't really compare the pre-processed output to figure out why.

```
patch/gfx9/chip/llpcGfx9Chip.cpp:849:26: error: use of undeclared identifier 'snprintf'; did you mean 'vswprintf'?
        int32_t length = snprintf(unknownRegNameBuf, 256, "UNKNOWN(0x%08X)", regId);
                         ^~~~~~~~
                         vswprintf
/usr/include/wchar.h:628:12: note: 'vswprintf' declared here
extern int vswprintf (wchar_t *__restrict __s, size_t __n,
           ^
patch/gfx9/chip/llpcGfx9Chip.cpp:849:35: error: cannot initialize a parameter of type 'wchar_t *' with an lvalue of type 'char [256]'
        int32_t length = snprintf(unknownRegNameBuf, 256, "UNKNOWN(0x%08X)", regId);
                                  ^~~~~~~~~~~~~~~~~
/usr/include/wchar.h:628:43: note: passing argument to parameter '__s' here
extern int vswprintf (wchar_t *__restrict __s, size_t __n,
                                          ^
2 errors generated.
```

Reply:
Huh, interesting. We do have one Clang-based builder internally that didn't show this. But yeah, it's obviously a reasonable change, I just got suspicious because it wasn't mentioned in the commit message.

Reply:
Yeah, sorry about that.  I was in auto-pilot ;)